#!/usr/bin/env python
PACKAGE = "velo2cam_calibration"

from math import pi

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("marker_size", double_t, 0, "Size of the marker (m)", 0.2, 0.1, 0.5)
gen.add("delta_width_qr_center", double_t, 0, "width increment from target center to qr center (m)", 0.55, 0, 1)
gen.add("delta_height_qr_center", double_t, 0, "height increment from target center to qr center (m)", 0.35, 0, 1)

exit(gen.generate(PACKAGE, "velo2cam_calibration", "Monocular"))
